index.html

```diff
@@ -167,14 +167,16 @@ <h2 class="languages-title">Skills</h2>
                 </section>
             </section>
             <section class="final-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</section>
-            <section class="data">
-                <input class="input-name" type="text" placeholder="Full name">
-                <input class="email-address" type="email" placeholder="Email address">
-                <textarea name="text1" class="text1" cols="30" rows="10">Enter text here</textarea>
-                <section class="side-im">
-                    <img class="side-form" src="images/form.svg" alt="side-form">
-                </section>
-                <button class="get-my-resume">Get in touch</button>
+            <section class="data">   
+                <form action="https://formspree.io/f/xayvoepp" method="post">
+                    <input class="input-name" type="text" placeholder="Full name" maxlength="30" required>
+                    <input class="email-address" type="email" placeholder="Email address" required>
+                    <textarea name="text1" class="text1" maxlength="500" cols="30" rows="10" required>Enter text here</textarea>
```

Comment:
-  [ ] Let's make sure that instead of using `Enter text here` as a value of the textarea to use `placeholder` for better user experience so when the user starts typing he doesn't have to erase the default value